install-develop.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -ex
+
+# realpath might not be available on MacOS
+script_path=$(python -c "import os; import sys; print(os.path.realpath(sys.argv[1]))" "${BASH_SOURCE[0]}")
+top_dir=$(dirname "$script_path")
+REPOS_DIR="$top_dir/third_party"
+BUILD_DIR="$top_dir/build"
+
+_check_submodule_present() {
+    if [ ! -f "$REPOS_DIR/$@/setup.py" ]; then
+       echo Didn\'t find $@ submodule. Please run: git submodule update --recursive --init
+        exit 1
+    fi
+}
+
+_check_submodule_present caffe2
+_check_submodule_present onnx
+
+if ! echo "$PATH" | grep ccache; then
```

Comment:
The directory path that contains ccache's cc, c++ ... commands don't necessary have "ccache" in it.

Reply:
Is there a better way to check this?

Reply:
Would `readlink $(which g++)` bet better?

Reply:
Yeah I think this will be better